src/components/KDropdown/KDropdown.vue

```diff
@@ -30,6 +30,7 @@
               v-if="triggerButtonText"
               :appearance="appearance"
               class="dropdown-trigger-button"
+              data-dropdown-trigger="true"
```

Comment:
if this isn't documented we're likely not going to know what it's for 😬

Reply:
Added a comment in KPop (not where the attribute is defined but rather where it's being used - I'm guessing someone who wants to find out what it's for will try searching for references in the project)